precompiles/utils/src/data.rs

```diff
@@ -37,6 +37,35 @@ impl From<Address> for H160 {
 	}
 }
 
+/// The `bytes`/`string` type of Solidity.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Bytes(pub Vec<u8>);
```

Comment:
IT may be better to remove inner pub

Reply:
It's main purpose it to wrap a `Vec<u8>` for encoding, and doesn't provide any guaranty on top of `Vec<u8>`, so it seems better to leave to ability to move out the vec and do things such as extending it.

Reply:
ok :)